src/lib/get-integration-badges.tsx

```diff
@@ -47,7 +47,16 @@ export function getIntegrationBadges(
 			break
 	}
 
+	let typeBadge: Badge
+	if (integration.integration_type) {
+		typeBadge = {
+			text: integration.integration_type.name,
+			tooltip: integration.integration_type.description,
+		}
+	}
+
 	return [
+		...(typeBadge ? [typeBadge] : []),
```

Comment:
I _think_ this ordering of `typeBadge` clashes with the intent behind `tierFirst` (which is to display the tier first)

Reply:
OH 🤦!  Great catch, I'll adjust!